### PR TITLE
project ready for marking

### PR DESCRIPTION
**Pull request by AkenEagle**

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+}
+
+/* Color variables */
+:root {
+    --black: #000000;
+    --darkGrey: #333333;
+    --light: #f5f5f7;
+    --red: #E50914;
+}
+
```
Comment (LeeFarnell2U) [inline assets/css/styles.css:14]: Not a bad thing, I just really like root variables so great job impletmenting this!

File: assets/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+/* For smooth scrolling when clicking links */
+html {
+    scroll-behavior: smooth;
+}
+
```
Comment (LeeFarnell2U) [inline assets/css/styles.css:19]: This was a lovely touch!

File: assets/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+/* Duration of transitions */
+#contact-me a, #about-me p, .first-project, .other-project {
+    transition-duration: 0.2s;
+}
+
+#contact-me a:hover, #about-me p:hover {
+    text-shadow: 0rem 0rem 0.7rem var(--red);
+}
```
Comment (LeeFarnell2U) [inline assets/css/styles.css:200]: Consider taking the hover of the `About Me` text. As per the feedback, consider only using hovers when elements do something e.g. a link or button.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+                <li>
+                    <a href="#contact-me">Contact me</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+
+    <!-- Banner area -->
+    <div class="banner">
+        <h2>My portfolio website</h2>
```
Comment (LeeFarnell2U) [inline index.html:33]: Consider changing this title. It's the main focal point of the site when you first visit so I think you would benefit more from having something more personal e.g. `Harmanjot Singh | Professional Portfolio` or something similar.

File: assets/css/styles.css (hunk 3 above)
Comment (AkenEagle) [reply assets/css/styles.css:200]: Yeah that's understandable, thanks for feedback! 👍

File: index.html (hunk 4 above)
Comment (AkenEagle) [reply index.html:33]: Yeah that's absolutely true, don't know how I missed something like that haha, thank you!

**Pull request merged**